### ISSUE TITLE
Unexpected git-tree-sha1 in get_default_args()

### ISSUE THREAD

**Issue by jonathanBieler** [JuliaInterop/Clang.jl#352]
Looks like the path to the include folder is wrong and this makes `get_default_args` fails. It also download the artifact every time which seems a bit strange.

v1.7, Clang.jl v0.15.0, macOS big sur (triple = "x86_64-apple-darwin14")

```
julia> args = get_default_args()
┌ Error: Tree Hash Mismatch!
│   Expected git-tree-sha1:   303e65f6896f295459c81aa311b5eb993f702d89
│   Calculated git-tree-sha1: 78a3cf7c8dfe0b0ad81b8ba0a86f7d89a8d19300
└ @ Pkg.Artifacts /Users/julia/buildbot/worker/package_macos64/build/usr/share/julia/stdlib/v1.7/Pkg/src/Artifacts.jl:364
ERROR: AssertionError: failed to setup environment due to missing dirs, please file an issue at https://github.com/JuliaInterop/Clang.jl/issues.
```

**Comment by Gnimuc** [comment]
Same on CI: https://github.com/JuliaInterop/Clang.jl/runs/4377467571?check_suite_focus=true

**Comment by Gnimuc** [comment]
https://github.com/JuliaLang/Pkg.jl/pull/2881

**Comment by Gnimuc** [comment]
Looks like this has been fixed in Julia 1.7.1.
